JPS_Ontology/ontology/ontogfa/ontogfa.csv

```diff
@@ -0,0 +1,8 @@
+Source,Type,Target,Relation,Domain,Range,Quantifier,Comment,Defined By,Label
+OntoGFA,TBox,http://www.theworldavatar.com/ontology/ontogfa/OntoGFA.owl,https://www.w3.org/2007/05/powder-s#hasIRI,,,,,,
+OntoGFA,TBox,1,http://www.w3.org/2002/07/owl#versionInfo,,,,,,
+OntoGFA,TBox,Calculation of gross floors area,http://www.w3.org/2000/01/rdf-schema#comment,,,,,,
+OntoGFA,TBox,,http://www.w3.org/2002/07/owl#imports,,,,,,
+Building,Class,,,,,,,,
+GFA,Data Property,,,,Float,,,,
+reference GFA,Data Property,,,,Float,,,,
```

Comment:
Please remove the ontology and create a separate pull request for this ontology.

Notably, there are plenty of missing information (definitions, domain and ranges). Also please follow the convention hasGFA and hasReferenceGFA for the data property. Use bot:building as the domain and do not create a new building concept.